cordapp-example/contracts-kotlin/src/main/kotlin/com/example/contract/CashContract.kt

```diff
@@ -18,14 +18,25 @@ class CashContract : Contract {
 
         when (command.value) {
             is Commands.Create -> requireThat {
-                // everyone can print/crate cash... because of covi... crisis
-                val cashState = tx.inputsOfType<CashState>().single()
-                "Cash value must be positive" using (cashState.balance > 0)
+                "There should be no input." using tx.inputsOfType<CashState>().isEmpty()
+                // in this exercise I allow anyone to create cash
```

Comment:
From the contract's point of view, everyone indeed can create cash. It's more for the users to decide why created cash is worth more than the other.